### ISSUE TITLE
Getting error of `Wrong server version.`

### ISSUE THREAD

**Issue by iseong83** [justanhduc/task-spooler#6]
Hi, I follow the installation instruction at Ubuntu18.04 and simply ran `ts`. But, I got the error message: `Wrong server version. Received 1048576, expecting 730`.
Could you help me to resolve this issue?

**Comment by justanhduc** [comment]
Hi @iseong83. Which version do you install, CPU or GPU? Are you able to reproduce it anywhere (for eg Colab)?
The error was caused by the miscommunication between server and client, but what caused it is beyond me at the moment.

**Comment by RathnaMurthy** [comment]
Hi,  I was following the installation instruction on Ubuntu20.04 and simply ran ts. But, I get the same error message: Wrong server version. Received -1035402781, expecting 730.
Could you also help me to resolve this issue?

**Comment by iseong83** [comment]
Hi @justanhduc. Thank you for your response. I installed the GPU version and I am using CUDA 10.2. I tested on Colab, it works well. I can not reproduce this issue in Colab.
 
You said it's because of the miscommunication between sever and client, do you mean the task-spooler server and client Can the pre-installed task-spooler in Ubuntu cause this issue? 
![image](https://user-images.githubusercontent.com/44355378/130646100-7a761056-6bb4-43d5-815a-bbe3b9ed79e3.png)

I also installed yours after removing the pre-installed one, but it still does not solve the issue. But, I wonder, I need to do something else to restart the server after installing yours.

**Comment by justanhduc** [comment]
@iseong83 Indeed! The problem is that `ts` tried to communicate and receive some packages from the server, but it got from `tsp` instead.
When you remove `tsp` without shutting down the server, the server is still there. To resolve this you should
1. Reinstall `tsp`
2. Run `tsp -K` to kill the server. 
3. Remove `tsp`
If you don't want to reinstall `tsp`, you can locate the server in the `/tmp` folder. It should have the format `socket-ts.{uid}`. Just remove it and you are done.
Let me know if there's still any problem.

**Comment by justanhduc** [comment]
Hi @RathnaMurthy. Which branch did you use to install? Could you please check whether you have another version of `tsp` like above?

**Comment by iseong83** [comment]
@justanhduc It works! I run `tsp -K`, but it seems there is still `socket-ts.{uid}` in `/tmp`. So I delete it also and it works now!
Thank you so much for helping me to solve this issue!
